scripts/3.TCGA-MLexample_Pathway.py

```diff
@@ -341,22 +357,23 @@ def get_threshold_metrics(y_true, y_pred, tissue='all'):
     y_tissue_pred_train = y_pred_train[y_train.index.isin(sample_sub)]
```

Comment:
If you created a dataframe with each observation and its training, cv, and testing predictions as well as its disease, this would be much cleaner?

The principle being never reorder or subset data separately when it can be performed jointly.